services/pipenv-helpers.js

```diff
@@ -16,6 +33,18 @@ const isLockfile = Joi.object({
   develop: Joi.object().pattern(Joi.string(), isDependency),
 }).required()
 
+/**
+ * Determines the dependency version based on the dependency type.
+ *
+ * @param {object} attrs - Refer to individual attributes
+ * @param {string} attrs.kind - Wanted dependency type, defaults to 'default'
```

Comment:
Lets note here that the other possible value is `'dev'`

Reply:
updated